nuxeo-runtime/nuxeo-runtime/src/main/resources/OSGI-INF/LoginAsComponent.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+
+<component name="org.nuxeo.runtime.LoginAsComponent" version="1.0">
+
+  <implementation class="org.nuxeo.runtime.api.login.LoginAsComponent" />
+
+  <service>
+    <provide interface="org.nuxeo.runtime.api.login.LoginAs" />
+  </service>
+
+  <extension-point name="impl">
```

Comment:
Please add a short `<documentation>` tag shortly describing the extension point.